Proposal: Manasi Danke-Amanda-Ezeobiejesi-Carla Duarte-Sylvia Lin

mmdanke opened the pull request.

amandaeze97 committed:
Added team members

mmdanke committed:
Update README.md

Commit:
Carla test

mmdanke committed:
Update and rename README.md to proposal.md

mmdanke committed:
Update proposal.md

mmdanke committed:
Create README.md

Commit:
Server methods

Fetching and updating user and room info

eos7l committed:
implement login.html

charlieroberts commented:
This sounds really really useful! I don't have much experience using Live25 so I'm unclear how much effort it will take to integrate your software with this system. Do you have a backup plan / alternative feature to implement if Live25 integration proves to be too challenging? If you already know this won't be a problem, then you're good to go!

eos7l committed:
implemented login

eos7l committed:
Merge pull request #1 from eos7l/stylingInit-Sylvia

Styling Init

eos7l committed:
fixed login

eos7l committed:
Merge pull request #2 from eos7l/stylingInit-Sylvia

fixed login

Commit:
Availability table

Commit:
Merge pull request #3 from eos7l/ciduarte/table

Availability table

Commit:
Table in template

mmdanke committed:
Edited Names of Buttons

mmdanke committed:
Edited HTML References

mmdanke committed:
Update reference.html

mmdanke committed:
Created register.html

mmdanke committed:
changed css for title

mmdanke committed:
Omitted logo, changed css for title

mmdanke committed:
Update register.html

mmdanke committed:
added window.onload part for const registerButton

mmdanke committed:
added potential register.js reference

mmdanke committed:
added potential register.js reference

mmdanke committed:
added potential register.js reference

mmdanke committed:
Updated register.html

Commit:
Fixed login

eos7l committed:
Table styling changes

Commit:
Bootstrap 4, integrate form

Commit:
Merge pull request #4 from eos7l/tableStyling-Sylvia

Table styling sylvia

mmdanke committed:
update register.html

mmdanke committed:
update register.js 

added path to login.html if registration parameters are properly entered

mmdanke committed:
Update register.js

Commit:
Merge pull request #5 from eos7l/mmdanke/registration

Mmdanke/registration

eos7l committed:
update form styling

Commit:
Merge remote-tracking branch 'origin/tableStyling-Sylvia' into tableStyling-Sylvia

Commit:
Merge header changes

Commit:
Merge pull request #6 from eos7l/tableStyling-Sylvia

Format header, created form

eos7l committed:
added room field

Commit:
Availability array

Commit:
Merge branch 'master' of https://github.com/eos7l/fp

Commit:
Form updates

Commit:
Merge pull request #7 from eos7l/ciduarte/separate-form-js

Updated form

Commit:
Added email

Commit:
🌧🐩 Checkpoint

Commit:
🌿💸 Checkpoint
./public/dashboard.html:2478586/1086

Commit:
🛍💇 Checkpoint
./public/dashboard.html:2478586/15

Commit:
🚨🛀 Checkpoint
./public/dashboard.html:2478586/441

Commit:
🐘🗽 Checkpoint
./public/dashboard.html:2478586/53

Commit:
👘📙 Checkpoint
./public/dashboard.html:2478586/445

Commit:
📙🐸 Checkpoint
./public/dashboard.html:2478586/189

Commit:
🛀🐾 Checkpoint
./public/dashboard.html:2478586/177
./public/css/style.css:2478586/63

Commit:
👗🚝 Checkpoint
./public/css/style.css:2478586/303
./public/dashboard.html:2478586/16

Commit:
✂️🌊 Checkpoint
./public/css/style.css:2478586/213
./public/dashboard.html:2478586/203

Commit:
📒🙈 Checkpoint
./public/dashboard.html:2478586/1096
./db.json:2478586/324

mmdanke committed:
Delete .glitch-assets

amandaeze97 committed:
Update db.json

Added all the rooms names.

Commit:
Quick Fixes

Commit:
Email functionality

Commit:
Fix header

Commit:
Valid room availability

Checks to see if the given room is available
at the selected times. If not, pops up a status
alert.

eos7l committed:
input field no longer clears

Commit:
Merge pull request #8 from eos7l/ciduarte/availability-validation

Valid room availability

eos7l committed:
update email form

eos7l committed:
Merge branch 'master' of https://github.com/eos7l/fp into updatedEmail

 Conflicts:
	public/dashboard.html

Commit:
Merge pull request #9 from eos7l/updatedEmail

Updated email

amandaeze97 committed:
Create meetingsDisplay.html

amandaeze97 committed:
Added meetings button

Adjusted column size for header,
Added meetings button

amandaeze97 committed:
Update dashboard.html

amandaeze97 committed:
Update meetingsDisplay.html

mmdanke committed:
Updating General Layout of README

amandaeze97 committed:
Made 17:30 to show up for startTime

amandaeze97 committed:
Made 17:30 to show up for startTime

Commit:
Quick Fixes

Email HTML was missing for some reason
Fixed DB to reflect user meetings

mmdanke committed:
Update proposal.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

Commit:
View meeting feature

Commit:
Register styling

Commit:
Updated README

eos7l committed:
Update README.md

Commit:
Merge pull request #10 from eos7l/amandaeze97-patch-1

Made 17:30 to show up for startTime

amandaeze97 committed:
Roominder logo and favicon

amandaeze97 committed:
Update README.md

amandaeze97 committed:
Added logo and favicon

amandaeze97 committed:
Added logo and favicon

amandaeze97 committed:
Added logo and favicon

amandaeze97 committed:
Added logo and favicon

Commit:
Logo

Commit:
Merge pull request #11 from eos7l/final-submission

Incorporated logo

Commit:
Update logo

Commit:
Merge pull request #12 from eos7l/final-submission

Update logo

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md

mmdanke committed:
Update README.md
